### ISSUE TITLE
Accessibility - My Profile Screen - Poor Color Contrast of Green Border

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-iOS#14]
The color contrast of the green border ( #5DA56B ) around my profile information and the white background ( #ffffff  )  is not. The color contrast ratio is 2.97:1 enough and needs to be 4.5:1 or higher to pass. Making the border a darker green or #3d3d3d (charcoal gray) will make it pass. 

Please see attached screenshot.

![safetynet - profile screen - poor color contrast of green border](https://user-images.githubusercontent.com/10423487/61545406-6d855300-aa15-11e9-9297-69155bd3137c.png)

**Comment by aanthonyDOL** [comment]
Made the background a darker green (#004D00)

**Comment by jfc3-dol** [comment]
Don't you mean the border and not background?

Because if the background is now dark green, the color contrast won't pass with black text, etc.
